app/models/manageiq/providers/kubernetes/container_manager/metrics_capture/hawkular_client_mixin.rb

```diff
@@ -1,4 +1,6 @@
 module ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCapture::HawkularClientMixin
+  attr_reader :client
```

Comment:
@yaacov instead of making `client` readable... why don't you define in this mixin the method `hawkular_verify_credentials`? It seems also better in term of code reuse.